packages/core/src/core.ts

```diff
@@ -682,7 +714,10 @@ class ThresholdKey implements ITKey {
     // acquireLock: false. Force push
     await this.addLocalMetadataTransitions({ input: [...authMetadatas, shareStore], privKey: [...sharesToPush, undefined] });
     if (delete1OutOf1) {
-      await this.addLocalMetadataTransitions({ input: [{ message: ONE_KEY_DELETE_NONCE }], privKey: [this.serviceProvider.postboxKey] });
+      await this.addLocalMetadataTransitions({
+        input: [{ message: ONE_KEY_DELETE_NONCE }],
+        privKey: [this.serviceProvider.postboxKey],
```

Comment:
is it possible the postboxkey is ed25519?

Reply:
Yes. For ed25519 the postbox key would also be ed25519.

Reply:
It would also be the scalar value.

Reply:
yes, that's correct. It ed25519 scalar